opaque_keys/edx/tests/test_block_usage_locators.py

```diff
@@ -26,9 +26,11 @@
 
 # Block usage locator to use in tests.
 TEST_ID_LOC = '519665f6223ebd6980884f2b'
-BLOCK_URL = 'block-v1:mit.eecs+6002x+2014_T2+{}@draft+{}@{}+{}@problem+{}@lab2'.format(
-    CourseLocator.BRANCH_PREFIX, CourseLocator.VERSION_PREFIX, TEST_ID_LOC,
-    BlockUsageLocator.BLOCK_TYPE_PREFIX, BlockUsageLocator.BLOCK_PREFIX
+BLOCK_URL = (
+    f'block-v1:mit.eecs+6002x+2014_T2+{CourseLocator.BRANCH_PREFIX}'
```

Comment:
Nit: Would have been nice to replace the specific course here with a test course locator.